### PR TITLE
Remove qwery from paid-containers.js

### PR DESCRIPTION
**Pull request by ealitten**
### What does this change?
Does this change need to be reproduced in dotcom-rendering ?
- [x] No
- [ ] Yes (please indicate your plans for DCR Implementation)

### What is the value of this and can you measure success?
Part of #23740

### Tested
- [x] Locally
- [ ] On CODE (optional)

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/paid-containers.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import bean from 'bean';
+import fastdom from 'fastdom';
+import { $$ } from '../../../lib/$$';
+
+const onKeyPress = (handler: EventListener) => (event: KeyboardEvent) => {
+	if (event.keyCode === 0x20 || event.keyCode === 0x0d) {
```
Comment (mxdvl) [inline static/src/javascripts/projects/commercial/modules/paid-containers.ts:6]: [`keyCode`] is deprecated, and we should probably be using [`code`][] instead. At the very least, it’d be useful to know what these codes relate to. I have no idea what `0x20` or `0x0d` are 🤷‍♂️.

[`keyCode`]: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

[`code`]: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code

**Review by mxdvl — APPROVED** [review]
Well done on converting to TypeScript. A small note on key codes, but otherwise good to go!

File: static/src/javascripts/projects/commercial/modules/paid-containers.ts (hunk 1 above)
Comment (mxdvl) [reply static/src/javascripts/projects/commercial/modules/paid-containers.ts:6]: It’s `Enter` and `Space` respectively. https://keycode.info/

`0x20` = `32` &rarr; `code`:`Enter`
`0x0d` = `13` &rarr; `code`:`Space`

Comment (MarSavar) [reply static/src/javascripts/projects/commercial/modules/paid-containers.ts:6]: Interesting, Max. I had a look at the documentation and it looks like [`key`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key) would also be a suitable (better?) alternative to [`code`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code) as it returns the value of the key being pressed (as opposed to the physical key on the keyboard).

Comment (mxdvl) [reply static/src/javascripts/projects/commercial/modules/paid-containers.ts:6]: In this case, as they’re modifier keys, I’d be inclined to use `key`.

File: static/src/javascripts/projects/commercial/modules/paid-containers.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import bean from 'bean';
```
Comment (mxdvl) [inline static/src/javascripts/projects/commercial/modules/paid-containers.ts:1]: There are rules that will prevent you from using `bean` in TypeScript files. You have to use `lib/$$` instead.

**Review by mxdvl — CHANGES_REQUESTED** [review]

File: static/src/javascripts/projects/commercial/modules/paid-containers.ts (hunk 2 above)
Comment (ealitten) [reply static/src/javascripts/projects/commercial/modules/paid-containers.ts:1]: Ah is this why teamcity build is breaking? It wasn't clear to me from the logs

File: static/src/javascripts/projects/commercial/modules/paid-containers.ts (hunk 1 above)
Comment (ealitten) [reply static/src/javascripts/projects/commercial/modules/paid-containers.ts:6]: Do you mean we should use `code` @mxdvl?

Comment (MarSavar) [reply static/src/javascripts/projects/commercial/modules/paid-containers.ts:6]: `code` returns the behaviour of a physical keyboard and might not work on a virtual keyboard or an alternative input device, according to [Google](https://developers.google.com/web/updates/2016/04/keyboardevent-keys-codes). 
I agree that `e.key === " "` isn't very readable but it does seem more accessible for people using a virtual keyboard that doesn't mimic the layout of a standard keyboard (I think I'm entering extreme edge case territory here...).
TLDR: both work!

Comment (mxdvl) [reply static/src/javascripts/projects/commercial/modules/paid-containers.ts:6]: I’m happy with either.

File: static/src/javascripts/projects/commercial/modules/paid-containers.ts (hunk 2 above)
Comment (arelra) [reply static/src/javascripts/projects/commercial/modules/paid-containers.ts:1]: You should see this rule highlighted in vscode.
There is a flakey test `scroll.spec.js` which fails the build sometimes

**Review by mxdvl — APPROVED** [review]
